examples/model_selection/plot_likelihood_ratios.py

```diff
@@ -152,20 +145,20 @@ def extract_score(cv_results):
 
 estimator = LogisticRegression()
 X, y = make_classification(n_samples=300, weights=[0.9, 0.1], random_state=0)
-_ = extract_score(cross_validate(estimator, X, y, scoring=scoring, cv=10))
+extract_score(cross_validate(estimator, X, y, scoring=scoring, cv=10))
 
 # %%
-# Plot class likelihood ratios
+# Invariance w.r.t. prevalence
 # ============================
```

Comment:
```suggestion
# Invariance with regards to prevalence
# =====================================
```
Let us help people who are not fluent in English, and might not know what "w.r.t." means